tests/bignum.egg

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Could you add an example where bigint and bigrat are stored as columns of some tables? The test here does not show what the name of the types are (`BigInt`, `BigRat`)